h/services/annotation_read.py

```diff
@@ -23,6 +24,42 @@ def get_annotation_by_id(self, id_: str) -> Optional[Annotation]:
         except InvalidUUID:
             return None
 
+    def get_annotations_by_id(
+        self, ids: List[str], eager_load: Optional[List] = None
+    ) -> Iterable[Annotation]:
+        """
+        Get annotations in the same order as the provided ids.
+
+        :param ids: the list of annotation ids
+        :param eager_load: A list of annotation relationships to eager load
+            like `Annotation.document`
+        """
+
+        if not ids:
+            return []
+
+        annotations = self._db.execute(
+            self._annotation_search_query(ids=ids, eager_load=eager_load)
+        ).scalars()
+
+        return sorted(annotations, key=lambda annotation: ids.index(annotation.id))
```

Comment:
~Is ordereding by ID not something postgres could do for us?~

Ah I get it now, sorted by the input ids :+1: